texar/torch/run/executor.py

```diff
@@ -901,6 +905,12 @@ def __init__(self, model: nn.Module,
             raise ValueError("Invalid value for `show_live_progress`")
         self._register_logging_actions(show_live_progress)
 
+        # tbx logging
+        if tbx_logging_dir is not None:
+            self.summary_writer = SummaryWriter(logdir=tbx_logging_dir)
+            self._tbx_logging_conditions = utils.to_list(tbx_log_every)
```

Comment:
Can we resort to `log_every` if `tbx_log_every` is `None`?

Reply:
Resolved in bbaf278